site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -210,72 +201,120 @@ export const WorkspaceScheduleForm: FC<WorkspaceScheduleFormProps> = ({
     { value: form.values.saturday, name: "saturday", label: Language.daySaturdayLabel },
   ]
 
+  const handleToggleAutoStart = async (e: ChangeEvent) => {
+    form.handleChange(e)
+    // if enabling from empty values, fill with defaults
+    if (!form.values.autoStartEnabled && !form.values.startTime) {
```

Comment:
are we checking `!form.values.startTime` here to ensure the user didn't fill something out previously?